runtime/testy/Ansible.py

```diff
@@ -0,0 +1,98 @@
+import socket
+import threading
+import time
+import fake_dawn
+import random
+import sys
+from runtimeUtil import *
+data = [0] #for testing purposes
+sendPort = 1235
+recvPort = 1236
+
+#Custom buffer for handling states. Holds two states, updates one and sends the other.
+class two_buffer(): 
+    def __init__(self):
+        self.data = [0, 0]
+        self.put_index = 0
+        self.get_index = 1
+    def replace(self, item):
+        self.data[self.put_index] = item
+        self.put_index = (self.put_index + 1) % 2
+        self.get_index = (self.get_index + 1) % 2
+    def get(self):
+        return self.data[self.get_index]
+
+sendBuffer = two_buffer()
+recvBuffer = two_buffer()
+raw_fake_data = [[0]]
+packed_fake_data = [[0]]
+dawn_buffer = [0]
+###Protobuf handlers###
+#Function for handling unpackaging of protobufs from Dawn
+def unpackage(data):
+    return data #will be changed when we fully support protos
+#Handles packaging and sending state to dawn in the form of protobuf we define
+def package(state, badThingsQueue):
+    s = "TEST"#will be changed when we fully support protos
+    b = bytearray()
+    b.extend(map(ord, s))
+    return b
+###Start Ansible Thread Chain###
+def packageData(badThingsQueue, stateQueue, pipe):
```

Comment:
This deserves a docstring that explains that this is run in a thread (or is it a process?)